tools/gen_stubs.zig

```diff
@@ -593,16 +624,19 @@ fn parseElf(parse: Parse, comptime is_64: bool, comptime endian: builtin.Endian)
 fn archIndex(arch: std.Target.Cpu.Arch) u8 {
     return switch (arch) {
         // zig fmt: off
-        .riscv64     => 0,
```

Comment:
I'm unsure if the the indexes that were assigned here previously had significance. If they did, I can revert this part and add the new arches at 10-12.